### DISCUSSION TITLE
BombSquad Mac App Store version

### DISCUSSION THREAD

**Discussion by ChaosCoder** [efroemling/ballistica#304] [Q&A]
Hey, I'm playing BombSquad occasionally since years on my Mac. Great game and so much fun!

However, the Mac App Store version of the game is still running on version 1.4.141 (2 years old) and was not updated since then.

Is there any plan to update that version? I was using my Game Center profile to sync my progress but on the "Test" builds you are providing this seems to not be possible to use. Is there another work around to get my profile with all the stuff I bought to the new "Test" version (1.6.4 currently)? Otherwise I would really appreciate if you could update the version in the Mac App Store 😊

Edit: Mentioning @efroemling, in order to raise awareness.

**Comment by Phong-Wind** [comment]
Me too. I've been looking forward to the new update on Mac App Store for a long time now. At first I thought BombSquad was discontinued from 2018 but then I found out this GitHub :)

**Reply by eduPro1pix** [reply]
I too want the same in snap store

**Comment by efroemling** [comment]
Sorry for the slow reply, but a quick explanation:
There are currently a number of outdated builds of the game out there (the Mac App Store, Oculus Rift, etc.).  Each one of these has some minor blocker that I need to take care of before I can update it (Mac App Store builds currently get rejected due to how the game handles hardware controllers, the Rift builds need to be updated to properly support touch controllers, etc). I do intend to get these all back up to date at some point, but for the time being I am iterating on the game and only releasing official updates on Android platforms just to keep things simple. The ballistica builds here as well as the bombsquad test builds at https://files.ballistica.net/bombsquad/builds/ should always be up to date though and hopefully enough to hold people over until I can get all store versions back in sync.

Worth mentioning: there was also a snap version in the Ubuntu store maintained by someone else, but it seems to have only been 1.4.x. If someone would like to put together a script to assemble a snap package from the repo here I'd be happy to start pushing updates to the store. I hope to do this myself when I get time but it's not at the top of my priority list.
